lib/std/zig/render.zig

```diff
@@ -2290,7 +2290,12 @@ fn renderToken(ais: *Ais, tree: Ast, token_index: Ast.TokenIndex, space: Space)
     const token_start = token_starts[token_index];
     const lexeme = tokenSliceForRender(tree, token_index);
 
-    try ais.writer().writeAll(lexeme);
+    // TODO remove this c_void -> anyopaque rewrite after the 0.10.0 release
+    if (mem.eql(u8, lexeme, "c_void")) {
+        try ais.writer().writeAll("anyopaque");
+    } else {
+        try ais.writer().writeAll(lexeme);
+    }
```

Comment:
Hmmm this is actually a *little* bit problematic because the following code will be incorrectly transformed:

```zig
const S = struct {
    c_void: i32,
    anyopaque: i32,
};
```

to

```zig
const S = struct {
    anyopaque: i32,
    anyopaque: i32,
};
```

causing a duplicate field compile error. `c_void` and `anyopaque` are primitives; not keywords, and so things such as block labels and field names are allowed to be named with those tokens.

Maaaaybe it's OK to let this slide?

But I think it would be wise instead to have a special handling for when the AST node is an identifier node and only do the transformation in such case.

Reply:
Good catch, probably would have been OK to let slide but I found a fairly clean way to do it and pushed a commit improving this to only rewrite identifier nodes.